Command.hs

```diff
@@ -0,0 +1,26 @@
+module Command
+    ( Command(..)
+    , parseCommand
+    ) where
+
+import Import hiding ((<>))
+import Options.Applicative
+
+data Command = RunServer | DeleteStaleComments
+
+parseCommand :: IO Command
+parseCommand = execParser $ info (helper <*> commandParser) fullDesc
+
+commandParser :: Parser Command
+commandParser = subparser
```

Comment:
Would this be easier to read as `subparser serverCommand <> deleteStaleCommentsCommand <|> pure RunServer`